system/services/presideObjects/SqlSchemaSynchronizer.cfc

```diff
@@ -190,17 +201,49 @@ component {
 			column.definitionSql = adapter.getColumnDefinitionSql( argumentCollection = args );
 			column.alterSql      = adapter.getAlterColumnSql( argumentCollection = args );
 			column.addSql        = adapter.getAddColumnSql( argumentCollection = args );
-			column.version       = Hash( column.definitionSql );
-
-			columnSql &= delim & column.definitionSql;
+			if( StructKeyExists( colMeta, "dbsync" ) and not colMeta.dbsync ) {
+				dbColumn = QueryFilter( columnsFromDb, function( col ) {
+					return col.column_name == colName;
+				});
+
+				if( dbColumn.recordcount ) {
+					dbArgs = {
+						tableName     = arguments.tableName
+						, columnName    = colName
+						, dbType        = dbColumn.type_name
+						, nullable      = dbColumn.is_nullable
+						, primaryKey    = dbColumn.is_primarykey
+						, autoIncrement = dbColumn.is_autoincrement
+						, maxLength     = adapter.doesColumnTypeRequireLengthSpecification( dbColumn.type_name ) ? ( IsNumeric( dbColumn.column_size ) ? dbColumn.column_size : 0 ) : 0
+					};
+
+					// Set sql based on current DB details to skip dbsync for this property
+					column.definitionSql = adapter.getColumnDefinitionSql( argumentCollection = dbArgs );
+					column.alterSql      = adapter.getAlterColumnSql( argumentCollection = dbArgs );
+					column.addSql        = adapter.getAddColumnSql( argumentCollection = dbArgs );
+				} else {
+					structDelete( sql.columns, colName );
+				}
+			} else {
+				columnSql &= delim & column.definitionSql;
+			}
+			column.version = Hash( column.definitionSql );
 			delim = ", ";
 		}
 
-
+		colMeta = "";
```

Comment:
I _think_ this variable is not `var`'d?

Reply:
Hi @DominicWatson ,

I've already declared this to var scope in line number 171 function name : _generateTableAndColumnSql() , please check